client/components/Application/Content.tsx

```diff
@@ -163,7 +163,6 @@ export const CONTROLLERS: {
   json: File,
   jsonschema: File,
   map: File,
-  package: File,
```

Comment:
I think it needs to be kept otherwise it will fail opening `datapackage.json`. Although better solution would be https://github.com/okfn/opendataeditor/issues/640 (cc @romicolman)